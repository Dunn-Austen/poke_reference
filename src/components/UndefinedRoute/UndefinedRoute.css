.undefined-route {
  align-items: center;
  background: grey;
  background: linear-gradient(rgba(0, 0, .5, .5), rgba(0, .1, .1, .1)), url('https://i.pinimg.com/originals/1d/a9/01/1da9012381ecbfbfdb455740ad29c9d8.jpg');
  background-blend-mode: lighten saturation;
  background-repeat: no-repeat;
  background-size: 100% 105%;
  display: flex;
  flex-direction: column;
  height: 93vh;
  justify-content: center;
  width: 100%
}

.surprised-pikachu {
  height: 300px;
  width: auto;
}

.return-home {
  font-size: 20px;
  height: 40px;
  margin-top: 20px;
  width: 150px;
}

.return-home:hover {
  background-color: gold;
}
