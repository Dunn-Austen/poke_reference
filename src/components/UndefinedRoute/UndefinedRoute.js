import React from 'react';
import { Link } from 'react-router-dom';
import './UndefinedRoute.css';

const UndefinedRoute = () => {
  return (
    <section className='undefined-route'>
      <img
        className='surprised-pikachu'
        src='https://i.kym-cdn.com/photos/images/original/001/431/201/40f.png'
        alt='iconic surprised pikachu face'
      />
      <Link to ='/'>
        <button className='return-home'>Return Home</button>
      </Link>
    </section>
  )
}

export default UndefinedRoute;
