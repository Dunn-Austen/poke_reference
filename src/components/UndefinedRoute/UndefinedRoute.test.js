import React from 'react';
import { shallow } from 'enzyme';
import UndefinedRoute from './UndefinedRoute';

describe('UndefinedRoute', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<UndefinedRoute />);
  });

  it('should match the UndefinedRoute Snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
