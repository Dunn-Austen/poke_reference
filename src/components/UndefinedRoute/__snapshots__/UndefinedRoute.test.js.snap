// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UndefinedRoute should match the UndefinedRoute Snapshot 1`] = `
<section
  className="undefined-route"
>
  <img
    alt="iconic surprised pikachu face"
    className="surprised-pikachu"
    src="https://i.kym-cdn.com/photos/images/original/001/431/201/40f.png"
  />
  <Link
    to="/"
  >
    <button
      className="return-home"
    >
      Return Home
    </button>
  </Link>
</section>
`;
